benches/sync_mpsc.rs

```diff
@@ -179,22 +163,39 @@ fn uncontented_unbounded(c: &mut Criterion) {
     });
 }
 
-criterion_group!(
-    create,
-    create_1_medium,
-    create_100_medium,
-    create_100_000_medium
-);
-
-criterion_group!(send, send_medium, send_large);
-
-criterion_group!(
-    contention,
-    contention_bounded,
-    contention_bounded_full,
-    contention_unbounded,
-    uncontented_bounded,
-    uncontented_unbounded
-);
-
-criterion_main!(create, send, contention);
+fn group_create_medium(c: &mut Criterion) {
+    let mut group = c.benchmark_group("create_medium");
+    create_medium::<1>(&mut group);
+    create_medium::<100>(&mut group);
+    create_medium::<100_000>(&mut group);
+    group.finish();
+}
+
+fn group_send(c: &mut Criterion) {
+    let mut group = c.benchmark_group("send");
+    send_data::<Medium, 1000>(&mut group, "Medium");
+    send_data::<Large, 1000>(&mut group, "Medium");
+    group.finish();
+}
+
+fn group_contention(c: &mut Criterion) {
+    let mut group = c.benchmark_group("contention");
+    contention_bounded(&mut group);
+    contention_bounded_full(&mut group);
+    contention_unbounded(&mut group);
+    group.finish();
+}
+
+fn group_uncontented(c: &mut Criterion) {
+    let mut group = c.benchmark_group("uncontented");
+    uncontented_bounded(&mut group);
+    uncontented_unbounded(&mut group);
+    group.finish();
+}
+
+criterion_group!(create, group_create_medium);
```

Comment:
I belive it's better that the function here starts wich `bench_` instead of `group_`(`group_create_medium` to `bench_create_medium `). Those functions are orginized by group, but they are benchmark function.